packages/devtools_app/lib/src/screens/inspector_v2/inspector_data_models.dart

```diff
@@ -459,21 +458,19 @@ class FlexLayoutProperties extends LayoutProperties {
 
     return FlexLayoutProperties._fromNode(
       node,
-      direction: _directionUtils.enumEntry(data['direction']) ?? Axis.vertical,
-      mainAxisAlignment: _mainAxisAlignmentUtils.enumEntry(
-        data['mainAxisAlignment'],
-      ),
-      mainAxisSize: _mainAxisSizeUtils.enumEntry(data['mainAxisSize']),
-      crossAxisAlignment: _crossAxisAlignmentUtils.enumEntry(
-        data['crossAxisAlignment'],
-      ),
+      direction: _directionNamesToValues[data['direction']] ?? Axis.vertical,
```

Comment:
can we use the `byName` lookup for all of these inspector ones instead of doing the asNameMap and then looking up by a key?

Reply:
I originally explored that for some of these, but `byName` throws if it can't find a matching value, so there's no opportunity to provide defaults like done here.